Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM arm64v8/node:17.3-bullseye
```

Comment:
I didn't use one because this will only be running on Pi devices for our pipeline. I can change it, but I didn't think it was necessary with our use case.